google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -79,7 +79,7 @@ def classify_stmt(query):
 
 def parse_insert(insert_sql, params):
     """
-    Parse an INSERT statement an generate a list of tuples of the form:
+    Parses an INSERT statement an generate a list of tuples of the form:
```

Comment:
Once again, we're usually writing in imperative style:
https://github.com/googleapis/python-spanner/blob/master/google/cloud/spanner_v1/transaction.py#L176
`parse`, not `parses`, here and everywhere